package.json

```diff
@@ -111,7 +111,9 @@
         }
       ],
       "@semantic-release/release-notes-generator",
-      "@semantic-release/npm"
+      "@semantic-release/npm",
+      "@semantic-release/changelog",
```

Comment:
I've actually been having trouble with the changelog one-- semantic release cant always find it? doesnt seem to be deprecated but still weird.

Anyways, according to the release this plugin makes changes to a changelog document, which this repo doesnt have. https://github.com/semantic-release/changelog

I think we can just remove this plugin.

Reply:
Even better this way :)

Reply:
Thank you! I removed it.